src/features/commands/poll.js

```diff
@@ -0,0 +1,145 @@
+                        .setDescription('Choose whether to allow multiple answers in the poll or not.')
+                        .setRequired(true))
+                .addStringOption(option =>
+                    option.setName('choice-a')
+                        .setDescription('First choice.')
+                        .setRequired(true))
+                .addStringOption(option =>
+                    option.setName('choice-b')
+                        .setDescription('Second choice.')
+                        .setRequired(true))
+                .addStringOption(option =>
+                    option.setName('choice-c').setDescription('Third choice.'))
+                .addStringOption(option =>
+                    option.setName('choice-d').setDescription('4th choice.'))
+                .addStringOption(option =>
+                    option.setName('choice-e').setDescription('5th choice.')))
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName('list')
+                .setDescription('Lists all polls that are created and are active in this server.'))
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName('results')
+                .setDescription('Displays results of a poll.')
+                .addStringOption(option => option.setName("poll-id").setDescription("The poll ID to get results from.").setRequired(true))),
+    async execute(client, interaction) {
+        const pollTitle = interaction.options.getString("title");
+        const pollChoices = [interaction.options.getString("choice-a"), interaction.options.getString("choice-b"), interaction.options.getString("choice-c"), interaction.options.getString("choice-d"), interaction.options.getString("choice-e")].filter((a) => a);
+
+        if (!interaction.guild) {
+            interaction.reply({ content: 'You can only use this command in servers!', ephemeral: true });
+            return;
+        }
+
+        switch (interaction.options._subcommand) {
+            case "create":
+                const multipleAnswersAllowed = interaction.options.getBoolean('allow-multiple-answers', true);
+                strawpoll.createStrawpoll(pollTitle, pollChoices, multipleAnswersAllowed).then(response => {
+                    db.createPoll(interaction.member.user.id, interaction.guild.id, `${pollTitle}:${response.pollId}`).then(resp => {
+                        const publicPollEmbed = new MessageEmbed()
+                            .setColor("#0099ff")
+                            .setTitle(`Poll: ${pollTitle}`)
+                            .setDescription(`You can vote [here](https://strawpoll.com/${response.pollId}).`)
+                            .setFooter(`Poll ID: ${response.pollId}`);
+
+                        interaction.reply({ content: `Poll "${pollTitle}" has been created successfully on this server.`, ephemeral: true });
+                        interaction.channel.send({ embeds: [publicPollEmbed] });
```

Comment:
Message is unnecessary and makes it impossible for moderators and users to figure who initiated it.

```suggestion
                        interaction.reply({ embeds: [publicPollEmbed] });
```